src/three/plugins/images/sources/GeoJSONImageSource.js

```diff
@@ -0,0 +1,423 @@
+import * as THREE from 'three';
+import { TiledImageSource } from './TiledImageSource.js';
+import { ProjectionScheme } from '../utils/ProjectionScheme.js';
+
+export class GeoJSONImageSource extends TiledImageSource {
+
+	constructor( {
+
+		geojson = null,
+		url = null, // URL or GeoJson object can be provided
+		tileDimension = 256,
+		bounds = [ - 180, - 90, 180, 90 ],
+		levels = 18,
+		projection = null, // 'EPSG:3857' or 'EPSG:4326' or null to auto- detect
```

Comment:
I think we can remove "projection" - the geojson spec specifies that [all values are in the WGS84 coordinate system](https://stevage.github.io/geojson-spec/#section-4). It sounds like it used to be the case that GeoJSON allowed for CRS specifications but I don't expect this to be common and I'd prefer to ask the user to convert their geojson to the common representation before passing it to the library for the sake of simplicity.